test/runtests.jl

```diff
@@ -3,15 +3,12 @@ using Test
 
 @testset "Expr" begin
     ex = :(f(a, b))
-    @test operation(ex) == :f
-    @test arguments(ex) == [:a, :b]
-    @test exprhead(ex) == :call
-    @test ex == similarterm(ex, :f, [:a, :b])
+    @test operation(ex) == :call
```

Comment:
operation here should be `f` now, right?

Reply:
Yes, sorry I haven't updated the tests yet.

Reply:
I will review when the tests are updated